src/components/ContactImportation/ContactImportationCard.jsx

```diff
@@ -0,0 +1,196 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Icon } from "cozy-ui/react";
+import IconCross from "../../assets/icons/small-cross.svg";
+import IconFileFailImport from "../../assets/icons/file-fail-import.svg";
+import IconFilePartialImport from "../../assets/icons/file-partial-import.svg";
+import IconFileVcf from "../../assets/icons/file-vcf.svg";
+import IconFileWrongFormat from "../../assets/icons/file-wrong-format.svg";
+import Importation from "../../importation";
+import Status from "../../importation/status";
+
+export default class ContactImportationCard extends React.Component {
+  render() {
+    const {
+      importation,
+      progress,
+      onFileSelected,
+      onFileUnselected
+    } = this.props;
+    const { report } = importation || {};
+    const { t } = this.context;
+
+    switch (importation.status) {
+      case Status.UNCONFIGURED:
+        return (
+          <CardWrapper clickable={true}>
```

Comment:
Instead of wrapping every single subtree by `<CardWrapper />` you created to respect DRY concepts, you should wrap the `switch/case` within the `<CardWrapper />` component:

```js
return <CardWrapper clickable={isClickable}>
  { switch(importation.status) {
      case Status.UNCONFIGURED:
        return <UnconfiguredCard />
      // etc.
    }
  }
  </CardWrapper>
```
With that refactoring, you can see how the architecture is strange: it is very unreadable to have a `switch/case` statement inside the JSX.

Another smell is the magic value for `clickable`: future developers could not understand why this value is sometime `true` and sometime `false`.